modules/adding-ibmz-lpar-agent.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_with_agent_based_installer/prepare-pxe-infra-agent.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="adding-ibmz-lpar-agents_{context}"]
+= Adding {ibm-z-title} Logical Partition (LPAR) as agents
+
+Use the following procedure to manually add {ibm-z-name} agents running in an LPAR. Use this procedure only for {ibm-z-name} clusters running in an LPAR.
+
+.Procedure
+
+. Create a boot parameter file for the agents.
++
+.Sample parameter file:
+[source,yaml]
+----
+rd.neednet=1 cio_ignore=all,!condev \
```

Comment:
Just checking if this is a YAML file or another type of file? The file structure here differs from our conventional YAML file.

Reply:
I agree this is a parm file I suggest to use` [source,terminal]`